x/pocketcore/types/cache.go

```diff
@@ -161,7 +165,8 @@ type SessionIt struct {
 func (si *SessionIt) Value() (session Session) {
 	err := ModuleCdc.UnmarshalJSON(si.Iterator.Value(), &session)
 	if err != nil {
-		panic(fmt.Errorf("can't unmarshal session iterator value into session: %s", err.Error()))
+		fmt.Println(fmt.Errorf("can't unmarshal session iterator value into session: %s", err.Error()))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("can't unmarshal session iterator value into session: %s", err.Error())
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)